src/main/java/zmq/io/net/tcp/TcpListener.java

```diff
@@ -174,14 +174,9 @@ public boolean setAddress(final String addr)
                 fd = options.selectorChooser.choose(address, options).openServerSocketChannel();
             }
 
-            //  IPv6 address family not supported, try automatic downgrade to IPv4.
-            if (fd == null && address.family() == StandardProtocolFamily.INET6 && options.ipv6) {
-                // TODO V4 automatic downgrade to IPV4
-            }
-            assert (fd != null);
-
             //  On some systems, IPv4 mapping in IPv6 sockets is disabled by default.
             //  Switch it on in such cases.
+            // Actually does nothing
```

Comment:
same comment here re: the "does nothing" comment

Reply:
again